src/components/truncate-long-text/truncateLongText.html

```diff
@@ -3,7 +3,11 @@
   styled directly in origin-web-common.  `truncated-content` is used by
   origin-web-console in certain contexts.
 -->
```

Comment:
Add a comment explaining highlightKeywords and linkify are mutually exclusive options.